doc/rst/usingchapel/multilocale.rst

```diff
@@ -198,16 +198,44 @@ ofi         slurm-srun
 =========   =============
 
 There are two ways to create co-locales. The first is to set the
-``CHPL_RT_LOCALES_PER_NODE`` environment variable. If its value is greater
-than one and less than or equal to the number of sockets on a node, then the
-Chapel runtime will run each locale in its own socket. The second way to
-create co-locales is to use the command-line argument ``-nl NxL``, where ``N``
-is the number of nodes and ``L`` is the number of locales per node. The ``L``
-is optional; if it isn't specified then Chapel will run the "ideal" number of
-locales based on the node architecture. Currently this is limited to the
-value of ``CHPL_RT_LOCALES_PER_NODE``; in future releases we plan to include
-more sophisticated heuristics such as automatically running one locale per
-socket on nodes with multiple sockets.
+``CHPL_RT_LOCALES_PER_NODE`` environment variable. If set, Chapel will run
+the specified number of locales per node. The second way to create co-locales
+is to use the command-line argument ``-nl NxLt``, where ``N`` is the number
+of nodes, ``L`` is the number of locales per node, and ``t`` is an optional
+suffix indicating the architectural feature to which the co-locales should be
+bound. The ``L`` is optional; if it isn't specified then Chapel will run
```

Comment:
Is there a different letter that can be used here other than `L`. Since `L` now means cache this could be confusing

Reply:
What if we change the cache suffix to something else such as 'a', the second letter in cache. That way it doesn't conflict with the 'L' above (which is documented in several places and makes sense to me as the number of locales on a node) and it isn't the only capitalized suffix.

Reply:
I think that would be ok, although `L` makes more sense than `a` to me.